src/layouts/Tutorial.tsx

```diff
@@ -193,6 +193,8 @@ export const TutorialLayout = ({
     frontmatter.postMergeBannerTranslation as TranslationKey | null
   const gitHubLastEdit = useClientSideGitHubLastEdit(relativePath)
   const intlLastEdit = "data" in gitHubLastEdit ? gitHubLastEdit.data! : ""
+  const useGitHubContributors =
+    frontmatter.lang === DEFAULT_LOCALE || crowdinContributors.length === 0
```

Comment:
Out of curiosity, is that ever going to happen? that we have an english file with crowdin contributors?

Reply:
Not to my knowledge... anything in English starts on GitHub, everything else through Crowdin.